src/pages/trips/plan.tsx

```diff
@@ -33,13 +57,13 @@ const Plan = () => {
             days: dateArray,
             destinations: destArray, 
             isPublic: formValues.isPublic,
-            profileId: 1,  
+            profileId: session.profile.id,  
         })
 
         router.push({
-            pathname: '/trips/[id]',
+            pathname: `/trips/${call.data.id}`,
```

Comment:
This may work, but this isn't the appropriate syntax for the 'pathname' field. The square brackets were in there for a reason - while they ultimately resolve to the same thing, the standard is to use the square brackets, whose value is replaced by the 'query' object.